core/dbt/cli/resolvers.py

```diff
@@ -0,0 +1,24 @@
+from pathlib import Path
+
+
+def default_project_dir():
```

Comment:
Again, this looks great but we can take advantage of some fancy pathlib stuff to make things easier on ourselves.

```
def default_project_dir():
    paths = list(Path.cwd().parents)
    paths.insert(0, Path.cwd()) # adds cwd to the list of paths
    return next((x for x in paths if (x / "dbt_project.yml").exists()), Path.cwd()) 
```

Reply:
@iknox-fa What is your reaction to this tweak? (Untested)
```
def default_project_dir():
    default_dir = Path.cwd()
    paths = list(default_dir).extend(default_dir.parents)
    return next((x for x in paths if (x / "dbt_project.yml").exists()), default_dir)
```